Add example_operations.gsc, slightly modify lgl_interpreter.py

paul891112 opened the pull request:
First draft of example_operations.gsc, checked and it works

modify do_class to return func(args[0],args[1:]) on line 700

Croxsy commented:
example gsc looks great thanks!  for what is the print statement in lgl. interpreter on line 403?

paul891112 commented:
> example gsc looks great thanks! for what is the print statement in lgl. interpreter on line 403?

Its a statement I added to test "define" but forgot to delete. I just deleted it.

Croxsy commented:
okay great, i think you can merge:)
